### ISSUE TITLE
Bug: Multi-Part Form Data raising when constructing OpenAPI schema

### ISSUE THREAD

**Issue by xiang12383** [starlite-api/starlite#363]
**code：**

```
from pydantic import BaseModel
from starlette.datastructures import UploadFile
from starlite import Body, post, RequestEncodingType, Starlite


class FormData(BaseModel):
    cv: UploadFile
    image: UploadFile

    class Config:
        arbitrary_types_allowed = True


@post(path="/file-upload")
async def handle_file_upload(
        data: FormData = Body(media_type=RequestEncodingType.MULTI_PART),
) -> None:
    ...


app = Starlite(route_handlers=[handle_file_upload])
```

**Error message**
```
(venv) PS C:\aha\someproj> uvicorn main:app --reload
INFO:     Will watch for changes in these directories: ['C:\\aha\\someproj']
INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [23368] using StatReload
Process SpawnProcess-1:
Traceback (most recent call last):
  File "C:\Users\aha\AppData\Local\Programs\Python\Python310\lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\aha\AppData\Local\Programs\Python\Python310\lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\aha\someproj\venv\lib\site-packages\uvicorn\_subprocess.py", line 76, in subprocess_started
    target(sockets=sockets)
  File "C:\aha\someproj\venv\lib\site-packages\uvicorn\server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "C:\Users\aha\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\aha\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 646, in run_until_complete
    return future.result()
  File "C:\aha\someproj\venv\lib\site-packages\uvicorn\server.py", line 67, in serve
    config.load()
  File "C:\aha\someproj\venv\lib\site-packages\uvicorn\config.py", line 479, in load
    self.loaded_app = import_from_string(self.app)
  File "C:\aha\someproj\venv\lib\site-packages\uvicorn\importer.py", line 21, in import_from_string
    module = importlib.import_module(module_str)
  File "C:\Users\haha\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\haha\someproj\.\main.py", line 64, in <module>
    app = Starlite(route_handlers=[handle_file_upload])
  File "pydantic\decorator.py", line 40, in pydantic.decorator.validate_arguments.validate.wrapper_function
  File "pydantic\decorator.py", line 134, in pydantic.decorator.ValidatedFunction.call
  File "pydantic\decorator.py", line 201, in pydantic.decorator.ValidatedFunction.execute
  File "C:\haha\someproj\venv\lib\site-packages\starlite\app.py", line 186, in __init__
    self.openapi_schema = openapi_config.create_openapi_schema_model(self)
  File "C:\haha\someproj\venv\lib\site-packages\starlite\config.py", line 283, in create_openapi_schema_model
    return construct_open_api_with_schema_class(schema)
  File "C:\haha\someproj\venv\lib\site-packages\pydantic_openapi_schema\utils\utils.py", line 68, in construct_open_api_with_schema_class
    schema_definitions = schema(schema_classes, by_alias=by_alias, ref_prefix=REF_PREFIX)["definitions"]
  File "pydantic\schema.py", line 144, in pydantic.schema.schema
  File "pydantic\schema.py", line 590, in pydantic.schema.model_process_schema
  File "pydantic\schema.py", line 631, in pydantic.schema.model_type_schema
  File "pydantic\schema.py", line 258, in pydantic.schema.field_schema
  File "pydantic\schema.py", line 536, in pydantic.schema.field_type_schema
  File "pydantic\schema.py", line 957, in pydantic.schema.field_singleton_schema
ValueError: Value not declarable with JSON Schema, field: name='cv' type=UploadFile required=True
```

**Comment by Goldziher** [comment]
Please give a valid code example so it can he reproduced

**Comment by Goldziher** [comment]
issue fixed in v1.11.0. please confirm
